initfiles/bin/init_roxie

```diff
@@ -40,9 +40,6 @@ fi
 export LIBC_FATAL_STDERR_=1
 export restarts=0
 
-ulimit -c unlimited
-ulimit -n $NUM_ROXIE_HANDLES
```

Comment:
It's ok for roxie/dafilesrv to set lower limits, if that's what the user asked for. But the default (if they didn;t set anything in particular) should be to leave at default.

But that can be a separate issue. I think should put back the configurable override for these components though